files/files.go

```diff
@@ -91,25 +50,33 @@ func (c Contents) Less(i, j int) bool {
 	return a.Destination < b.Destination
 }
 
-func (c *Content) WithFileInfoDefaults() {
-	if c.FileInfo == nil {
-		c.FileInfo = &ContentFileInfo{}
+func (c *Content) WithFileInfoDefaults() *Content {
```

Comment:
this method could be called by multiple threads at once, therefore we probably need to modify and return a copy instead.

Reply:
👍